src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -205,8 +223,12 @@ namespace OpenMS
     static constexpr const char* INTEGRATION_TYPE_SIMPSON = "simpson";
     /// Baseline type: base to base
     static constexpr const char* BASELINE_TYPE_BASETOBASE = "base_to_base";
-    /// Baseline type: vertical division
+    /// Baseline type: vertical division (min of end points)
```

Comment:
Maybe we should specify that this is kept for some backward compatibility, to not confuse future readers.

Reply:
@pasqoo do you need this for backwards compatibility or can we just remove it?